server/routerlicious/packages/services-core/src/messages.ts

```diff
@@ -145,4 +150,18 @@ export interface IControlMessage {
 export enum ControlMessageType {
     // Instruction sent to update Durable sequence number
     UpdateDSN = "updateDSN",
+
+    // Instruction sent to have deli nack all future messages
+    NackFutureMessages = "nackFutureMessages",
+}
+
+export interface IUpdateDSNControlMessageContents {
+    durableSequenceNumber: number;
+    clearCache: boolean;
+}
+
+export interface INackFutureMessagesControlMessageContents {
+    code: number;
+    type: NackErrorType;
+    reason: string;
 }
```

Comment:
Can't we use the "INackContent" interface in protocol.ts? Or extend it?

Reply:
Yea we can reuse it